frontend/src/game/effects/LightEffect.ts

```
@@ -0,0 +1,81 @@
+    this.destroy();
+    const objects = this.parseLightObjects(tilemapKey);
+    objects.forEach((context) => {
+      const LightClass = registry[context.class];
+      if (!LightClass) return;
+      const light = new LightClass(this.scene, context);
+      this.lights.push(light);
+    });
+  }
+
+  destroy(): void {
+    this.lights.forEach((l) => l.destroy());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)